pom.xml

```diff
@@ -70,6 +70,11 @@
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
```

Comment:
In Spring Boot projects, it's usually better to opt for starters rather than individual dependencies (look for spring-boot-starter-validation)